python/runtime/pai/submitter.py

```diff
@@ -130,15 +132,18 @@ def get_oss_model_url(model_full_path):
     return "oss://%s/%s" % (oss.SQLFLOW_MODELS_BUCKET, model_full_path)
 
 
+def parse_maxcompute_dsn(datasource):
```

Comment:
Do we need this method or just write `MaxComputeConnection.get_uri_parts(datasource)` when calling?

Reply:
Good, idea. I will figure this out and do some modify if possible. In next PR :)